src/api/myndlaApi.ts

```diff
@@ -68,7 +72,7 @@ export const fetchSingleTopic = async (
   page: number | undefined,
   pageSize: number | undefined,
   context: Context,
-): Promise<ITopicWithPosts> => {
+): Promise<GQLArenaTopicV2> => {
```

Comment:
Vi må gå vekk fra å bruke backend typen for NodeBB kallene inntil vi vet om vi skal videre utvikle myndla arena eller ikke. Så jeg fikk gaute til å gjøre det som en løsningn inntil vi har bestemt oss 👯

Reply:
Som vi snakket om så stemmer jeg da for å lage en konverteringsfunksjon sånn at vi ikke bare caster backend json'en til GQL-typene. Tipper det blir mye bomming av det :grimacing: